scripts/mea/et/internal_witness.py

```diff
@@ -292,6 +312,10 @@ def process_verifier_notes(self):
                         if spec_func in edge['source']:
                             edge['note'] = self.process_comment(note)
                             break
+        
+        for key_name_env in action_type_data:
+            self._edges.append(action_type_data[key_name_env][0])
+            self._edges.append(action_type_data[key_name_env][1])
```

Comment:
There is no order of edges here. `Enter` - add before the current edge, `Return` - after the current edge